arangod/StorageEngine/TransactionState.h

```diff
@@ -74,6 +74,10 @@ class TransactionState {
     virtual ~Cookie() = default;
   };
 
+  static bool SortServerIds(ServerID const& lhs, ServerID const& rhs) {
```

Comment:
I still disagree with the function name, but it is not important.
Let's leave it as it is now.

Reply:
ah I only fixed the struct. Sorry